content/help/releases/2019-005.md

```diff
@@ -0,0 +1,155 @@
+
+This will be reflected on the original discussion by setting the canonical URL of the discussions pages to point to the article. Additionally a notice will displayed to the user indicating the link.
+
+![image](https://user-images.githubusercontent.com/1770056/55844936-aa37d900-5b0d-11e9-8555-1f4be2635708.png)
+
+### Knowledge Base Sitemaps
+
+Sites making use of the Sitemaps plugin will now offer an index of all knowledge base articles for search engines.
+
+### Filter by Knowledge Base or Community Category in Search
+
+The `/kb/search` page now include the ability to filter by a single knowledge base while searching knowledge base content and the ability to filter by a category while searching the community.
+
+![image](https://user-images.githubusercontent.com/1770056/55845613-9772d380-5b10-11e9-8b8b-3bc6b60d0127.png)
+
+### Improved Security with CSP
+
+Knowledge base pages now implement `Content-Security-Policy` headers. These headers prevent the loading of scripts from unauthorized script sources. Some domains (such as the site's own domain) are whitelisted by default.
+
+Additional domains may be whitelisted by extending the `ContentSecurityPolicy.ScriptSrc.AllowedDomains` array or implementing a `ContentSecurityPolicyProviderInterface` and registering it with `ContentSecurityPolicyModel`.
+
+### Improved Help Center home page
+
+The home page for Help Center type knowledge bases has been redesigned to make better use of screen space and improve user experience.
+
+![image](https://user-images.githubusercontent.com/1770056/55845961-f422be00-5b11-11e9-9b65-2e2e34e6ece2.png)
+
+### Improved Knowledge Base Editor
+
+The knowledge base editor now includes an improved text editor UI, that collapses when scrolling to leave more space for content.
+
+![image](https://user-images.githubusercontent.com/1770056/55846050-4d8aed00-5b12-11e9-8300-eba0bdf8b4ba.png)
+
+### Improved UI Cohesiveness
+
+This release saw numerous improvements to overall UI cohesiveness and polish. Areas of significant improvement include
+
+-   Form inputs
+-   Buttons
+-   Modals
+-   General text-alignment & metrics
+
+### Google Sign In 
+
+Due to the fact that **[Google is sunsetting Google+ Sign In](https://support.google.com/plus/answer/9217723?hl=en)** method we are adding Google Sign In. Forums that are using Google+ now are strongly recommended to create an OpenID OAuth 2.0 application at Google and turn on the Google Sign In plugin.
+
+For more information about the user authentication workflow that is being implemented [please read this documentation on Google.com](https://developers.google.com/identity/protocols/OpenIDConnect). For instructions on how to set up the plugin see [Vanilla user docs for Google Sign In](https://docs.vanillaforums.com/help/sso/social-connect/#google)
+
+![image](https://user-images.githubusercontent.com/2651070/55909483-30096200-5baa-11e9-8c2c-d11a9118c13d.png);
+
+## Other Fixes
+
+### Developer Notes
```

Comment:
Why are these in the Developer Notes section?

And where the actual developer notes that I wrote?